examples/react-native/src/Price.js

```diff
@@ -1,15 +1,15 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import { InstantSearch } from 'react-instantsearch/native';
+import { InstantSearch } from 'react-instantsearch-native';
 import {
   connectRefinementList,
   connectSearchBox,
   connectRange,
   connectMenu,
-} from 'react-instantsearch/connectors';
-import Stats from './components/Stats';
+} from 'react-instantsearch-core';
```

Comment:
We should import from `react-instantsearch-native`. The `core` package is only used internally.